layers/meta-balena-5x-owa/recipes-core/switchwifi/files/switchwifi.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Systemd services used for powering up Wifi/BLE module on boot using Owasys tools
+After=pmsrv.service network-pre.target
+Wants=pmsrv.service network-pre.target
+Requires=network-pre.target
+DefaultDependencies=no
+         
+[Service]   
+Type=oneshot 
+RestartSec=10     
+Restart=on-failure
+StartLimitBurst= 2                
+ExecStart=/usr/bin/Start_BT_WiFi 1
```

Comment:
This is already done in `resin-init` so the `switchwifi` recipe is no longer needed.

Reply:
Moved to [PR 28](https://github.com/balena-os/balena-owa5x/pull/28)